sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/implementation/models/TableServiceStats.java

```diff
@@ -6,8 +6,10 @@
 
 import com.azure.core.annotation.Fluent;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
 
 /** Stats for the service. */
+@JacksonXmlRootElement(localName = "StorageServiceStats")
 @Fluent
 public final class TableServiceStats {
```

Comment:
Was there a reason to use `Stats` instead of `Statistics` used in the other Storage SDKs?

Reply:
It's a generated class so it looks like this is how the service folks specified the model should be named.